docs/2.7.0/docs/canton/usermanual/domains/ethereum.rst

```diff
@@ -202,7 +202,7 @@ Canton supports mutual TLS between Canton and Ethereum client nodes and the demo
 configure this.
 Concretely, the TLS configuration for Canton expects a key store and the path to the Ethereum TLS certificates:
 
-.. literalinclude:: /canton/includes/mirrored/enterprise/app/src/pack/examples/e03-ethereum-sequencer/canton-conf/simple/tls.conf
+.. literalinclude:: /canton-drivers/examples/e03-ethereum-sequencer/canton-conf/simple/tls.conf
```

Comment:
Will there be also an update to the canton repository?

Specifically, will these example be deleted from the canton repository?

If you need help with that removal, I can help.

Reply:
i already have a PR where i remove the examples (but the pipeline line fails because the docs cant be generated without the). That is why im here :) 

Thanks for offering

Reply:
i will make sure to update it accordingly aswell then. it is not only the examples being deleted, it is much more (associated driver's aswel)

Reply:
There is another "problem". The RST files with the canton directory in this repo are used by the canton repository to build a "canton documentation" (verification). This happens also on CircleCI for canton.

With the current changes, e.g. `literalinclude:: /canton-drivers/examples/e03-ethereum-sequencer/canton-conf/simple/tls.conf`, the canton build will fail.

To fix that, either
* the canton-drivers RST files need to move outside of the canton directory in this repository, or 
* I will have to modify the canton build to somehow ignore these changes.

Reply:
hmm okay, so it actually makes more sense to move the `ethereum.rst` and `fabric.rst` domain usermanual out into it's own folder for canton-drivers ?

I was not aware of this tight coupling between the files

Reply:
The canton build should not be putting undue constraints on the docs repo.